src/fu-util-common.c

```diff
@@ -144,18 +146,20 @@ fu_util_print_device_tree (GNode *n, gpointer data)
 	g_autofree gchar *tmp = NULL;
 	g_auto(GStrv) split = NULL;
 
-	/* root node */
-	if (dev == NULL) {
-		g_print ("○\n");
-		return FALSE;
-	}
```

Comment:
I'm not sure I get this bit; if the device is NULL it's because it's the root node. There's no point trying to call `fu_util_device_to_string()` on NULL.